cmd/serve/httplib/httplib.go

```diff
@@ -1,4 +1,6 @@
 // Package httplib provides common functionality for http servers
+//
+// Deprecated: httplib has been replaced with lib/http
```

Comment:
Can we remove the whole `cmd/server/httplib` directory now? If so we should instead of deprecating it.

Reply:
I was going to refactor the other dependant services in separate PRs to keep this one smaller. httplib can't be deleted as it sits.

Reply:
Can we delete the unused code from it?

Ideally we'd do this in the same commit as moving it to somewhere else but that might be tricky now!

Reply:
I don't believe there is any unused code. This code also hasn't been moved. Are you referring to something else?

Reply:
I think I got confused!